docs/README.md

```diff
@@ -0,0 +1,33 @@
+# Docs Directory
+
+## How-Tos
+* [Dev Of The Week Zendesk](dev-of-the-week-zendesk.md) - How to find stuff in zendesk if you are dev of the week
+* [Dropbox Share Guide](dropbox-share-guide.md) - How to add new dropbox shares to staging
+* [Honeybadger](honeybadger.md) - How to use honeybadger to triage site issues
+* [Importing Data into Pegasus](importing-data.md)
+* [Instrumenting with Google Analytics](instrumenting-analytics.md)
+* [Logging](logging.md) - How to do various things with dashboard and pegasus logs
+* [Where Are The Logs](where-are-the-logs.md) - Some more information about logging
```

Comment:
@islemaster should this doc get combined with what's in [logging.md](logging.md)?